src/transform/groovy/ga/ozli/minecraftmods/groovylicious/transform/config/ConfigASTTransformation.groovy

```diff
@@ -101,7 +101,7 @@ class ConfigASTTransformation extends AbstractASTTransformation {
                 // Looks like the @Mojo/@Mod is in a different file, let's register a transform to the
                 // GroovyliciousMojoTransformRegistry to add a static { configDataClass.init() } to the Mod's main class
                 if (DEBUG) println "Adding transform to @GroovyliciousMod"
-                GroovyliciousMojoTransformRegistry.addTransform { AnnotationNode modAnnotation, ClassNode modClass ->
+                GModASTTransformer.registerTransformer { ClassNode modClass, AnnotationNode modAnnotation, SourceUnit source$ ->
```

Comment:
Is there still a need for the GroovyliciousMojoTransformRegistry or does GModASTTransformer.registerTransformer replace it?